types/tabulator-tables/index.d.ts

```diff
@@ -285,7 +285,7 @@ export interface OptionsPagination {
     paginationInitialPage?: number | undefined;
 }
 
-export type GroupArg = string | string[] | ((data: any) => any);
+export type GroupArg = string | string[] | ((data: any) => any) | Array<(data: any) => any> | Array<string | (data: any) => any>;
```

Comment:
The second last entry is a duplicate of the more general form in the last entry, so it can go.